uint8 PASSIVE = 0
uint8 ENABLE = 1

uint8 mode
uint8 num
float64 speed
float64 hz
time stamp
